docs/cache/manage/operations.mdx

```diff
@@ -0,0 +1,65 @@
+---
+sidebar_label: Measuring Usage (Operations)
+sidebar_position: 9
+title: Momento Cache measuring usage with operations
+pagination_next: null
+description: Learn how Momento Cache meters and bills for usage based on operations.
+hide_title: true
+---
+
+# Momento Cache measures usage with operations
+
+## What is an operation?
+
+An operation is a unit of work that reflects the utilization of networking and compute power. As a request consumes more resources, the operation count will increase proportionally.
+
+The number of operations incurred by a request depends on two factors:
+- the size of user data transfered in the API call
+- the API called, plus any parameters that modify its behavior
+
+## Calculating operation cost of an API call
+
+The base operation count for an API call is the number of 1024 byte chunks in _each_ item or element transfered by the request and by the response.
+
+When a request or response contains no items or elements (for example, a `BatchGet` request contains a list of _keys_), the user-supplied data it contains is treated as a single item.
+Protocol overhead is not counted towards request and response data size.
+
+Each API has an operational complexity factor, as listed in the table below. The total operation count for an API call is the complexity factor multiplied by the base operation count.
+
+All validly-authenticated API calls, including empty results or misconfigured requests, incur a minimum of 1 operation.
+
+## API Complexity Factors
+This table defines the operational complexity factors for Momento Cache APIs. If an API is not listed below, its complexity factor is 1.
+
+<table className="customTable">
+  <thead>
+    <tr>
+      <th>API Name</th>
+      <th>Complexity Factor</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr><td>SetIfAbsent</td><td>2</td></tr>
+    <tr><td>SetIfPresent</td><td>2</td></tr>
+    <tr><td>SetIfEqual</td><td>2</td></tr>
+    <tr><td>SetIfNotEqual</td><td>2</td></tr>
+    <tr><td>SetIfPresentAndNotEqual</td><td>2</td></tr>
+    <tr><td>SetIfAbsentOrEqual</td><td>2</td></tr>
+  </tbody>
+</table>
+
+
+## API Parameter Modifiers
+
+Some APIs may be modified by the [ReadConcern parameter](/cache/learn/how-it-works/read-concern). When `ReadConcern` is set to `Consistent`, the operation cost is multiplied by an additional factor of 2.
+
+
+## Examples
+
+`Get` for one item with a 16-byte key that does not exist incurs 1 operation, because the response will be empty: `CEIL(16 / 1024) + CEIL(0 / 1024)`
+
+`GetBatch` for one hundred 1-byte elements with 16-byte keys incurs 102 operations: `CEIL(100 * 16 / 1024) + 100 * CEIL(1 / 1024)`
+
+`SetIfAbsent` has a complexity factor of 2. Writing one 1500-byte item with a 16-byte key incurs 5 operations: `CEIL(16 / 1024) + 2 * CEIL(1500 / 1024)`
```

Comment:
I think I'm not getting how the "complexity factor" is applied.
Here the complexity factor of 2 is applied to the 1500-byte item but for `DictionaryGetFields` the complexity factor of 2 for read consistency is applied to both 10-byte items and 16-byte keys.